Moki/client/create_dev_rpm.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/sh
+
+# go to a temp directory we checked out the git repository
+# set as WORKDIR
+
+#DIR=Moki/client
+REP=$branch_name
+NAME=moki-client-dev
+SPEC=$NAME.spec
+RPMARCH=x86_64
+
+# use il package.json
+ln -sf package.json package-intuitive.json
+
+#cd $DIR
+
+#FIXES
+ls -la
```

Comment:
remove this please, we don't need to list the content, it was just for debugging purposes